core/azure-core/src/main/java/com/azure/core/polling/PollResponse.java

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.polling;
+
+/**
+ * The life cycle of an operation depicted below
+ * NOT-STARTED --- IN-PROGRESS
+ * ------ Successfully Complete
+ * ------ User Cancelled
+ * ------ Failed
+ **/
```

Comment:
Let's do a push sometime soon (once we're API complete) on JavaDoc. Discuss with me about the expectations before you start.